src/app/pages/services/components/service-rsync/rsync-configure/rsync-configure.component.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef, Component, OnInit,
+} from '@angular/core';
+import { Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@ngneat/reactive-forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import helptext from 'app/helptext/services/components/service-rsync';
+import { RsyncConfigUpdate } from 'app/interfaces/rsync-config.interface';
+import { EntityUtils } from 'app/pages/common/entity/utils';
+import { FormErrorHandlerService } from 'app/pages/common/ix-forms/services/form-error-handler.service';
+import { WebSocketService, DialogService, AppLoaderService } from 'app/services';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-rsync-configure',
+  templateUrl: './rsync-configure.component.html',
+  styleUrls: ['./rsync-configure.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class RsyncConfigureComponent implements OnInit {
+  form = this.fb.group({
+    port: [873, Validators.pattern('^[0-9]*$')],
```

Comment:
the last thing, it may be worth considering adding min and max validators to define the allowed port range. For example
https://github.com/truenas/webui/blob/master/src/app/pages/system/general-settings/gui-form/gui-form.component.ts#L39-L40